master/text.html

```diff
@@ -2472,7 +2472,7 @@ <h3 id='TextShapeMargin'>The <span class="property">'shape-margin'</span> proper
     </tr>
     <tr>
       <th>Value:</th>
-      <td><a>&lt;length&gt;</a> | <a>&lt;percentage&gt;</a></td>
+      <td><a>&lt;length-percentage&gt;</a></td>
```

Comment:
This is still `<length> | <percentage>` in CSS Shapes, but it probably just hasn't been updated yet, or should it remain as it is?

Reply:
Probably should be updated there, too.